libs/tree-sitter-wing/grammar.js

```diff
@@ -396,13 +396,13 @@ module.exports = grammar({
       ),
 
     new_expression: ($) =>
-      seq(
+      prec.right(seq(
         "new",
         field("class", choice($.custom_type, $.mutable_container_type)),
-        field("args", $.argument_list),
+        field("args", optional($.argument_list)),
```

Comment:
does this mean `new cloud.Bucket;` is now valid in Wing? Feels like we should add an example Wing file or mention it in the spec if that's the case

If this is just an optimization for better parsing or better diagnostics, then let's add a comment here to clarify that arguments aren't actually optional

Reply:
No, our parser will catch it. I mean technically it could work but there is currently an intentional error.

I can add a comment for this and other similar ones (e.g. `a.b.` is valid as well)